test/35-system-info.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { SinonStub, stub } from 'sinon';
+import * as systeminformation from 'systeminformation';
+
+import { getMemoryInformation } from '../src/lib/system-info';
+import { expect } from './lib/chai-config';
+
+function kbToMb(kilobytes: number) {
+	return Math.floor(kilobytes / 1024);
+}
+
+describe('System information', function () {
+	it('Returns the correct memory values', async () => {
+		// Stub the output of the systeminformation module
+		stub(systeminformation, 'mem').resolves({
+			total: 763472 * 1024,
```

Comment:
can we clean up some of these magic numbers since they seem to be used in multiple places? it's a little painful on the eyes and hopefully will prevent any fat fingering down the road.

Reply:
Yeah, good point, my original idea was to mock the output of `/proc/meminfo` hence the magic numbers.